rest/tests/test_rest.py

```diff
@@ -0,0 +1,570 @@
+"""
+Unit tests for REST api
+"""
+
+from __future__ import unicode_literals
+
+from rest_framework.status import (
+    HTTP_200_OK,
+    HTTP_400_BAD_REQUEST,
+    HTTP_403_FORBIDDEN,
+    HTTP_404_NOT_FOUND,
+    HTTP_405_METHOD_NOT_ALLOWED,
+)
+
+from learningresources.models import Repository
+from taxonomy.api import create_vocabulary, create_term
+from taxonomy.models import Vocabulary, Term
+from rest.serializers import (
+    RepositorySerializer,
+    VocabularySerializer,
+    TermSerializer,
+)
+from .base import (
+    RESTTestCase,
+    REPO_BASE,
+    API_BASE,
+    as_json,
+)
+
+
+class TestRest(RESTTestCase):
+    """
+    REST test
+    """
+
+    def test_repositories(self):
+        """
+        Test for Repository
+        """
+        repositories = self.get_repositories()
+        self.assertEqual(1, repositories['count'])
+
+        self.create_repository()
+        repositories = self.get_repositories()
+        self.assertEqual(2, repositories['count'])
+        new_repo_slug = repositories['results'][1]['slug']
```

Comment:
The repositories do come back in order by `id`, but I'm fine renaming this to `repo_slug`